guard/src/commands/validate.rs

```diff
@@ -65,7 +65,8 @@ or rules files.
         } else if app.is_present("last-modified") {
             last_modified
         } else {
-            regular_ordering
+            // making default alphabetical
+            alpabetical
```

Comment:
Can we simplify the if...else to always default to "alphabetical" unless "last-modified" is present? Line #63 seems like an unnecessary explicit check adding another branch.